package.json

```diff
@@ -25,6 +26,7 @@
   "scripts": {
     "start": "while(true) do node standupbot.js; sleep 5; done",
     "test": "mocha",
+    "migrations": "knex migrate:latest",
     "lint": "eslint ."
   },
   "engine": "node >= 0.8.14"
```

Comment:
@borcean was going to go through ``standupbot`` and clean up its lint as good js practice. I'm trying to write good js, but I'm going to leave adding ``"use strict";`` to him since that's a good thing to know about.